pynq/lib/_pynq/_audio/audio_adau1761.cpp

```diff
@@ -309,12 +309,17 @@ extern "C" void deselect(int iic_index) {
  * @return  none.
  *****************************************************************************/
 extern "C" void bypass(unsigned int audio_mmap_size,
-                       unsigned int nsamples, 
+                       unsigned int nsamples, unsigned int volume,
                        int uio_index, int iic_index) {
     int i, status;
     void *uio_ptr;
     int DataL, DataR;
     int iic_fd;
+    
+    if(volume > 63) {
```

Comment:
This check is not necessary, since you are checking it in the python API already.